ext/winevt/winevt_utils.cpp

```diff
@@ -267,7 +267,12 @@ extract_user_evt_variants(PEVT_VARIANT pRenderedValues, DWORD propCount)
         rb_ary_push(userValues, rbObj);
         break;
       case EvtVarTypeHexInt64:
-        rbObj = rb_sprintf("%#I64x", pRenderedValues[i].UInt64Val);
+        uint32_t  high;
+        uint32_t  low;
+
+        high = pRenderedValues[i].UInt64Val >> 32;
+        low = pRenderedValues[i].UInt64Val & 0x00000000FFFFFFFF;
+        rbObj = rb_sprintf("0x%04x%08x", high, low);
```

Comment:
Why higher part is `04x` length?

Reply:
Hmm, we should consider using hex length....

Reply:
I also used a fixed length for high-order bit in https://github.com/fluent-plugins-nursery/winevt_c/pull/35/commits/0c4c1be0c3d6ffd01e5fcfa3998d67ffd054fa17.